### ISSUE TITLE
Add option to save the translations history

### ISSUE THREAD

**Issue by runo280** [crow-translate/crow-translate#89]
Add option to save the translations history, so we can import that in some apps like [Anki](https://apps.ankiweb.net/docs/manual.html#importing)
I thinks it will be very useful.

**Comment by Shatur95** [comment]
Yes, I already have a proposal for translations history, the ability to save it will be added too.

**Comment by guissmo** [comment]
This would be a very good addition. Is there any way to help?

**Comment by Shatur** [comment]
> This would be a very good addition. Is there any way to help?

Feel free to send a pull request. I will happy to review it and draft a new release.

**Comment by mojvat** [comment]
we need a history of translated word.
please add

**Comment by Shatur** [comment]
> we need a history of translated word.

Do not bump the issue without significant information.
I currently busy, but anyone interested in it could send a PR, I will review it.
